### DISCUSSION TITLE
(nitpick) Example sentence for 雪崩 could be reworded a bit

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#54] [Errors and Corrections]
JP: 雪崩で道路がふさがった。

EN: A snowslide obstructed the road.

---

This is a huge nitpick but while the sentence and its translation are technically correct, the active vs passive voice is changed. It's normal in Japanese to express things in passive voice that would otherwise be expressed in active voice in English, so I am not entirely against this translation, but I think this sentence even in English would flow well in passive voice. Making it closer to the Japanese would be more beneficial although it's a relatively minor change.

"The road was obstructed by a snowslide"

**Comment by stephenmk** [comment]
The latest versions of Jitendex contain links to the JMdict and Tatoeba databases within each entry. So for example, there's a link to [this Tatoeba sentence](https://tatoeba.org/en/sentences/show/141957) in the entry for 雪崩.

<details>
<summary>Entry for 雪崩</summary>

![nadare](https://github.com/stephenmk/Jitendex/assets/8003332/615c3c18-dee7-4ab4-a11c-8e39251bd8a8)
</details>

If you'd like, you can add new translations yourself. It's the same amount of work for me either way (I still have to go change the ID indexed to JMdict to the value of the newly added sentence), but you would receive attribution on Tatoeba for the new sentence instead of me. If that's not something you care about, that's fine too.

**Reply by Morgawr** [reply]
Thanks for the pointer, I've never contributed to Tatoeba upstream but I made a new account and I'll start contributing more there as well. I noticed it's possible to have multiple variants in the same language for the same sentence, so I added this variant https://tatoeba.org/en/sentences/show/12271141 which more closely resembles the Japanese version.

**Reply by stephenmk** [reply]
Thanks, I'll update the index to use your new sentence. Speaking of nitpicking, I think they always want you to end sentences with a period or some other form of punctuation on Tatoeba.

**Reply by Morgawr** [reply]
Agh you're right, not sure how I missed it. Fixed, thanks!
